src/index.js

```diff
@@ -295,7 +295,11 @@ function ncc (
       libraryTarget: esm ? 'module' : 'commonjs2',
       strictModuleExceptionHandling: true,
       module: esm,
-      devtoolModuleFilenameTemplate: sourceMapBasePrefix + '[resource-path]'
+      // if prefix is `../`, then final path results in `.././`
+      // if prefix is `../../`, then final path results results in `../.././`
+      // which ideally would have been normalized by webpack, but alas it isnt,
+      // so we need to adjust sourceMapBasePrefix
+      devtoolModuleFilenameTemplate: sourceMapBasePrefix.slice(0, -1) + '[resource-path]'
```

Comment:
```suggestion
      devtoolModuleFilenameTemplate: sourceMapBasePrefix.endsWith('/') ? sourceMapBasePrefix.slice(0, -1) : sourceMapBasePrefix + '[resource-path]'
```

This should be conditional right? What if you accidentally remove the filename?

Reply:
Actually lets go with

```js
sourceMapBasePrefix.replace(/\/$/, '') + '[resource-path]'
```

Reply:
```suggestion
      devtoolModuleFilenameTemplate: sourceMapBasePrefix.replace(/\/$/, '') + '[resource-path]'
```

Reply:
@styfle applied the change, pls re-review